components/add_groups_to_channel_modal/add_groups_to_channel_modal.test.jsx

```diff
@@ -86,9 +89,9 @@ describe('components/AddGroupsToChannelModal', () => {
         setTimeout(() => {
             expect(instance.handleResponse).toBeCalledTimes(2);
         }, 0);
-        expect(instance.handleHide).not.toBeCalled();
+        expect(instance.handleHide).toHaveBeenCalledTimes(1);
```

Comment:
So the author of the tests initial intention was correct. The `instance.handleHide` should not be called. The reason it's still being called is because they've incorrectly initialized above `linkGroupSyncable`. If we fix that, then the test should fall in line with the authors intentions.